website/pages/en/index.js

```diff
@@ -52,10 +59,10 @@ const Logo = props => (
 )
 
 const ProjectTitle = props => (
-  <h2 className="projectTitle">
-    {siteConfig.title}
-    <small>{siteConfig.tagline}</small>
-  </h2>
+  <div>
```

Comment:
Is a `div` really necessary here? Or did you add it because of Reacts need to enclose element in one container, could you use `React.Fragment` instead?

Reply:
Yes, I will replace it with React.Fragment :)